tensilelite/Tensile/LibraryIO.py

```diff
@@ -251,7 +253,20 @@ def parseLibraryLogicFile(filename, archs=None):
 def parseLibraryLogicData(data, srcFile="?", archs=None):
     """Parses the data of a library logic file."""
     if isinstance(data, List):
+        # TODO: this can be removed when all logic files have dict format
         data = parseLibraryLogicList(data, srcFile)
+    else:
+        libraryType = data["LibraryType"]
```

Comment:
I wonder if here on the `else` branch, we should convert it into an `elif` and check against the expected type (Dict) and the library logic version?

Reply:
Thanks, good point, I will have it check the `LibraryLogicVersion` and type